CLAUDE.md

```
@@ -319,6 +319,6 @@ GitHub Actions workflows in `.github/workflows/`:
 
 ## Useful Links
 
-- Documentation: https://ferriskey.rs/docs/welcome/introduction
+- Documentation: https://docs.ferriskey.rs/getting-started/introduction
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)